_transactions.md

```diff
@@ -1080,3 +1083,53 @@ Besides the limits listed there, it's also recommended to preemptively [check a
   We currently do not expose an endpoint that lists all the limits that would apply for a given transaction.
   Your application should be prepared to handle these limit errors and forward the relevant information to the user so they can adjust the transaction properties accordingly.
 </aside>
+
+## Transacting with credit/debit cards
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox transactions. These can be used to force specific outcomes and test the various flows pertaining to using card data to perform transactions.
+
+<aside class="notice">
+  Any value entered for <i>expiry date</i> and <i>CVV</i> will be accepted as valid when used in conjunction with the card data listed in this section.
+</aside>
+
+### Credit/debit card deposits and withdrawals
+
+The following list contains specific transaction amount values that are meant to trigger a target error when trying to perform deposits or withdrawals on any given card account.
+
+Amount | Error                 |
+------ | --------------------- |
+12.12  | card_unauthorized     |
+14.14  | card_declined_by_bank |
+18.18  | request_data_invalid  |
+20.20  | card_expired          |
+22.22  | card_unsupported      |
+26.26  | insufficient_funds    |
+28.28  | amount_invalid        |
+34.34  | velocity              |
+50.50  | invalid               |
+
+<aside class="notice">
+  The <code>invalid</code> error is a catch-all that is issued when an unexpected error code is returned from a provider.
+</aside>
+
+### 3DS test cards
```

Comment:
Should these be on the account section? 🤔

Reply:
They are now, as per: https://github.com/uphold/docs/pull/241#discussion_r1455575059.